webapp/web/configuration.css

```diff
@@ -153,11 +124,44 @@
     padding: 2px;
 }
 
-.configure-package__tag-responses-item:focus {
+.tag-responses__item:focus {
     outline-color: #ddd;
 }
 
-.configure-package__button-add-responses-action {
+.tag-responses__add-tag-modal {
```

Comment:
I think we can remove the `tag_responses` prefix here and just use `add-tag-modal`, or even just `modal`? that way you can use the BEM strategy for the elements of the modal (`modal__heading`, modal__dropdown` etc.)

Reply:
Changed it